src/components/Nav/Nav.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Nav.css';
+
+export default function Nav() {
+  return (
+    <nav>
+      <NavLink
+        to="/rgb/255/0/0"

```

Comment:
This is a cool way to adjust the link color! Nice work.

Reply:
thanks, found this after looking up ways to check if something is active in react. found here https://reactrouter.com/en/main/components/nav-link